sdk/purview/azure-analytics-purview-sharing/CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Release History
```

Comment:
Missing [samples/README.md](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/anomalydetector/azure-ai-anomalydetector/src/samples/README.md), [test-resources.json](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/anomalydetector/azure-ai-anomalydetector/test-resources.json), [tests.yml](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/anomalydetector/azure-ai-anomalydetector/tests.yml).
Can be added in a follow-up PR once we have the ci.yml and pom.xml in the repo for pipelines to be configured first.

Reply:
We discussed addition of these changes at a later point due to complexity of test resource configuration.